provider/go.mod

```diff
@@ -9,15 +9,15 @@ require (
 	github.com/golang/protobuf v1.5.3
 	github.com/moby/buildkit v0.11.4
 	github.com/moby/moby v23.0.1+incompatible
-	github.com/moby/patternmatcher v0.5.0
 	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.47.3
-	github.com/pulumi/pulumi/pkg/v3 v3.66.0
-	github.com/pulumi/pulumi/sdk/v3 v3.66.0
-	github.com/stretchr/testify v1.8.2
+	github.com/pulumi/pulumi-terraform-bridge/v3 v3.50.1
+	github.com/pulumi/pulumi/pkg/v3 v3.71.0
```

Comment:
Would be best if the updates were a separate (automated) PR.

Reply:
I can run that first if you prefer

Reply:
https://github.com/pulumi/pulumi-docker/pull/669